app/Services/WorklogService.php

```diff
@@ -35,6 +36,18 @@ public function getLoggedInUserWorklogs(): Collection
         return $worklogs;
     }
 
+    public function getAllWorklogs(): Collection
+    {
+        $worklogs = $this->worklog->orderByDesc('created_at')->get();
```

Comment:
can be written as `$this->worklog->latest()->get()` . view elloquent documentation for more of such helper methods.

Reply:
Since yr using user detail as well in index view. View the document of eager loading and try to implement it